src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -160,6 +160,10 @@ private async Task CreateBlocksAsync()
                 catch (OperationCanceledException)
                 {
                 }
+                catch (ConsensusErrorException ce) when (ce.ConsensusError.Code == "invalid-collateral-amount")
```

Comment:
@quantumagi ah this feel so "hacky" still. But for now it will have to do. Can you please add a TODO above this line so that we can look at handling this better? (We need to rerun the build anyway)

Reply:
Done.